onnxruntime/python/tools/transformers/onnx_exporter.py

```diff
@@ -267,11 +295,8 @@ def export_onnx_model_from_pt(model_name, opset_version, use_external_data_forma
                               input_names, use_gpu, precision, optimize_onnx, validate_onnx, use_raw_attention_mask, overwrite,
                               model_fusion_statistics):
 
-    config = AutoConfig.from_pretrained(model_name, cache_dir=cache_dir)
-    if hasattr(config, 'return_dict'):
-        config.return_dict = False
-
-    model = load_pretrained_model(model_name, config=config, cache_dir=cache_dir, custom_model_class=model_class)
+    config, model = load_pt_model(model_name, model_class, cache_dir)
+    # config, model = load_pt_model_from_tf(model_name)
```

Comment:
Could you add a unit test for load_pt_model_from_tf?

Reply:
added a test for tf2pt_pipeline